tests/slenkins/slenkins_node.pm

```diff
@@ -75,6 +75,8 @@ sub run {
           .= "zypper -n --no-gpg-checks in " . join(' ', split(/[\s,]+/, get_var('SLENKINS_INSTALL'))) . "\n";
     }
 
+    my $firewallservice = check_var('VERSION', '15') ? 'firewalld' : 'SuSEFirewall2';
```

Comment:
In general this is a bad practice as we have to revisit it for '15-SP1'. Better use `utils::sle_version_at_least '15'`

Reply:
Thanks, good point. Another issue is that 'SuSEFirewall2' service doesn't exist, it should be 'SuSEfirewall2', fixing

Reply:
fixed here https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3927